doc/Learn/Mvux/Advanced/Messaging.md

```diff
@@ -280,18 +286,18 @@ public partial record PeopleModel
 }
 ```
 
-The `SelectedPersonPhone` state will only be refreshed if it falls under the predicate criteria, which is limited to the currently selected `Person`.
+The `SelectedPersonPhone` state will only be refreshed if it meets the predicate criteria, which are limited to the currently selected `Person`.
 
 > ![NOTE]
 > The `Selection` method above picks up UI selection changes and reflects them onto a state. This subject is covered [here](xref:Uno.Extensions.Mvux.Advanced.Selection).
 
 - `Observe<TOther, TEntity, TKey>(IState<TEntity> state, IFeed<TOther> other, Func<TOther, TEntity, bool> predicate, Func<TEntity, TKey> keySelector)`
 
-This overload is the same as the previous one, except it watches a single-item state rather than a list-state, as in the previous example.
+This overload is the same as the previous one, except it watches a single-item state rather than a `ListState,` as in the last example.
```

Comment:
```suggestion
This overload is the same as the previous one, except it watches a single-item state rather than a `ListState`, as in the last example.
```